cmd/utils.go

```diff
@@ -2113,3 +2114,107 @@ func checkValidLicense(log *LoggingConfig, license string) error {
 	}
 	return errors.New("file must have a valid license ID, see https://spdx.org/licenses/ for the list of valid licenses")
 }
+func lintMountPathForSingleFile(path string, log *LoggingConfig) {
+
+	file, err := os.Stat(path)
+	if err != nil {
+		log.Warning.logf("Could not stat mount path: %s", path)
+
+	} else {
+		if file.Mode().IsDir() {
+			log.Debug.logf("Path %s for mount is a directory", path)
+		} else {
+
+			log.Warning.logf("Path %s for mount points to a single file.  Single file Docker mount paths cause unexpected behavior and will be deprecated in the future.", path)
+		}
+
+	}
+}
+
+func lintMountVar(mountListSource string, log *LoggingConfig, stackPath string) (int, int) {
```

Comment:
These functions will only be used in `stack_lint_dockerfile_stack.go` so I think they should be moved there.